src/clojure/clojang/jinterface/otp/messaging.clj

```diff
@@ -126,26 +187,27 @@
     (.hashCode this))
   (link [this recip-pid]
     (.link this recip-pid))
-  (ping [this node-name timeout]
-    (.ping this timeout))
   ;; XXX put the ping with the default timeout in the mid-level API
-  (ping [this node-name]
-    (.ping this 1000))
+  (ping
```

Comment:
This is a potentially breaking change (though I think not). I've collected the `ping` clauses into a multi-arity protocol method, mostly so `core.typed` would stop complaining, but also it follows the pattern set by `exit` et al.